packages/cli/lib/ast-helper.js

```diff
@@ -141,3 +141,74 @@ function isPrimaryKeyFlag(idInitializer) {
 
   return false;
 }
+
+exports.getDataSourceConfig = function (fileContent) {
+  const ast = tsquery.ast(fileContent);
+  // Find a top-level declaration of `config` variable
+  const configVarDecl = tsquery(
+    ast,
+    // It must be a top-level declaration, not inside a function
+    'SourceFile>VariableStatement>VariableDeclarationList>' +
+      // The declaration should be for property called `config`
+      'VariableDeclaration:has([name="config"])',
```

Comment:
I'm a bit confused; does it have to be called `config` or does it have to include the word `config`?

Reply:
I see now that the comment on line 152 is confusing.

The variable must be called exactly `config`, as in:

```ts
const config = // ...
```

Reply:
Let me explain in more detail.

`VariableDeclaration` node has two children:
 - `Identifier` describing the variable name,
 - `ObjectLiteralExpression` describing the variable value when it's an object.

The query `VariableDeclaration:has([name="config"])` is matching `VariableDeclaration` that has a child node with `name="config"`.

You can try https://tsquery-playground.firebaseapp.com to build a better understanding. The playground provides visualization of TypeScript AST tree for any code snippet you provide and can run queries too.

Reply:
Ah okay, I see. Thank you for the explanation and link!